boxyhq/jackson
# Prefill redirect url setup link

## What does this PR do?



Admin will fill name, description and redirect Urls along with tenant & product while creating setup link for SSO Connection.

Fixes #814 

## Type of change




## How should this be tested?



- Create a new Setup link for SSO.
- Use that link to create SAML & OIDC
- Perform operations like update connection from Setup link page & admin page
- Verify if the operations are working as expected

## Checklist:
